::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 10 Aug 2017 18:32:37 +0200
Subject: [PATCH] classes: baremetal: arm-pod3 config hardcode

Populate MaaS configuration with node information for arm-pod3.

FIXME: Read all this info from PDF (Pod Descriptor File) later.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 .../baremetal-mcp-ocata-odl-ha/infra/maas.yml      | 40 +++++++++++-----------
 .../baremetal-mcp-ocata-ovs-ha/infra/maas.yml      | 40 +++++++++++-----------
 2 files changed, 40 insertions(+), 40 deletions(-)

diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml
index 5649990..e4765ca 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml
@@ -48,57 +48,57 @@ parameters:
       machines:
         kvm01:
           interface:
-            mac: "00:25:b5:a0:00:2a"
+            mac: "e0:ff:f7:00:08:ae"
           power_parameters:
-            power_address: "172.30.8.75"
-            power_password: "octopus"
+            power_address: "10.0.8.3"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         kvm02:
           interface:
-            mac: "00:25:b5:a0:00:3a"
+            mac: "e0:ff:f7:00:08:b1"
           power_parameters:
-            power_address: "172.30.8.65"
-            power_password: "octopus"
+            power_address: "10.0.8.4"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         kvm03:
           interface:
-            mac: "00:25:b5:a0:00:4a"
+            mac: "e0:ff:f7:00:08:bd"
           power_parameters:
-            power_address: "172.30.8.74"
-            power_password: "octopus"
+            power_address: "10.0.8.5"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         cmp001:
           interface:
-            mac: "00:25:b5:a0:00:5a"
+            mac: "e0:ff:f7:00:08:c6"
           power_parameters:
-            power_address: "172.30.8.73"
-            power_password: "octopus"
+            power_address: "10.0.8.6"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         cmp002:
           interface:
-            mac: "00:25:b5:a0:00:6a"
+            mac: "e0:ff:f7:00:08:cf"
           power_parameters:
-            power_address: "172.30.8.72"
-            power_password: "octopus"
+            power_address: "10.0.8.7"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
   linux:
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml
index 73215ee..26115fe 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml
@@ -48,57 +48,57 @@ parameters:
       machines:
         kvm01:
           interface:
-            mac: "00:25:b5:a0:00:2a"
+            mac: "e0:ff:f7:00:08:ae"
           power_parameters:
-            power_address: "172.30.8.75"
-            power_password: "octopus"
+            power_address: "10.0.8.3"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         kvm02:
           interface:
-            mac: "00:25:b5:a0:00:3a"
+            mac: "e0:ff:f7:00:08:b1"
           power_parameters:
-            power_address: "172.30.8.65"
-            power_password: "octopus"
+            power_address: "10.0.8.4"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         kvm03:
           interface:
-            mac: "00:25:b5:a0:00:4a"
+            mac: "e0:ff:f7:00:08:bd"
           power_parameters:
-            power_address: "172.30.8.74"
-            power_password: "octopus"
+            power_address: "10.0.8.5"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         cmp001:
           interface:
-            mac: "00:25:b5:a0:00:5a"
+            mac: "e0:ff:f7:00:08:c6"
           power_parameters:
-            power_address: "172.30.8.73"
-            power_password: "octopus"
+            power_address: "10.0.8.6"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
         cmp002:
           interface:
-            mac: "00:25:b5:a0:00:6a"
+            mac: "e0:ff:f7:00:08:cf"
           power_parameters:
-            power_address: "172.30.8.72"
-            power_password: "octopus"
+            power_address: "10.0.8.7"
+            power_password: "admin"
             power_type: ipmi
             power_user: "admin"
-          architecture: 'amd64/generic'
+          architecture: 'arm64/generic'
           distro_series: xenial
           hwe_kernel: hwe-16.04
   linux:
